src/markdown-pages/install-with-kurl/removing-object-storage.md

```diff
@@ -0,0 +1,67 @@
+---
+path: "/docs/install-with-kurl/removing-object-storage"
+date: "2021-12-17"
+weight: 21
+linktitle: "Removing Object Storage"
+title: "Removing Object Storage Dependencies"
+---
+
+As of [v2021.12.20-0](https://kurl.sh/release-notes/v2021.12.20-0), kURL clusters can be installed without add-ons having a peer dependency on object storage.
+There is also a migration path for clusters to remove object storage. This eliminates the need for either the Rook or MinIO add-ons.
+
+# New Installations
+
+The following add-ons can now be installed with or without object storage:
+1. **Registry**: Without an object store in the installer spec, a persistent volume (PV) will be used for storage. So if the Rook or MinIO add-on is installed, the Registry will always be backed by that object store.
+1. **Velero**: Without object storage in the spec, no default BackupStorageLocation will be created. If the `disableS3` flag is set to `true` for the KOTS add-on, a PV-backed storage location will be created as the default location using the [Local-Volume-Provider](https://github.com/replicatedhq/local-volume-provider) plugin.
+1. **KOTS**: If the `kotsadm.disableS3` flag is set to `true` in the installer, KOTS will be deployed without an object store. This will deploy KOTS as a StatefulSet using a persistent volume (PV) for storage. It will also disable the use of MinIO for hostpath and NFS snapshot storage destinations.
+
+This installer spec is an example of deploying a new cluster without using any object storage.
+```yaml
+apiVersion: cluster.kurl.sh/v1beta1
+kind: Installer
+metadata:
+  name: no-object-storage
+spec:
+  kubernetes:
+    version: 1.21.x
+  containerd:
+    version: 1.4.x
+  weave:
+    version: 2.6.5
+  longhorn:
+    version: 1.2.x
+  registry:
+    version: 2.5.7
+  velero:
+    version: 1.7.x
+  kotsadm:
+    version: 1.58.x
+    disableS3: true
+```
+
+# Cluster Migrations
+
+The migration path to remove object storage dependencies and components involves two steps.
+
+## Setting `disableS3` to `true` in the KOTS Add-On
+
+Object storage can be removed from some of the add-ons in the cluster by simply setting the `kotsadm.disableS3` flag in the KOTS addon to `true`.
+See the documentation in the [KOTS add-on](/docs/add-ons/kotsadm) for more information on the `disableS3` flag.
+
+When you re-install or upgrade using the updated installer spec (see the previous section for a sample), you should expect:
```

Comment:
@DanStough What section is considered the "previous section for an example"? Are you referring to the YAML file under the New Installations section? It would be best to give a link to the specific topic/section you're referring to and some context. For example, you could say "When you re-install or upgrade using the updated installer spec (see the New Installations section for a YAML example), you should expect:..."

And add the link to the text "New Installations section"